Source/Scene/Scene.js

```diff
@@ -1444,7 +1493,7 @@ define([
 
         if (environmentState.isSunVisible) {
             environmentState.sunDrawCommand.execute(context, passState);
-            if (scene.sunBloom) {
+            if (scene.sunBloom && !scene._useWebVR) {
```

Comment:
>Do you think this is a bug in Cesium?

Most likely. It was working in the PR for refactoring the scene to make it easier for multiple viewports, but it probably broke after changes from review. I'll add it to the VR/Renderer roadmap.